### PR TITLE
adds analytics code

### PR DESCRIPTION
**Pull request by matthewcarleton**
This code got dropped at some point so I'm adding it back in so we can properly track this site.

### CODE REVIEW

File: website/layouts/partials/scripts.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+{{ else }}
+{{ $js := $js | minify | fingerprint }}
+<script src="{{ $js.RelPermalink }}" integrity="{{ $js.Data.Integrity }}" crossorigin="anonymous"></script>
+{{ end }}
+<script type="text/javascript">
+    if (("undefined" !== typeof _satellite) && ("function" === typeof _satellite.pageBottom)) {
+        _satellite.pageBottom();
+    }
+</script>
+{{ partial "hooks/body-end.html" . }}
```
Comment (camilamacedo86) [inline website/layouts/partials/scripts.html:21]:
```suggestion
{{ partial "hooks/body-end.html" . }}

```

Comment (camilamacedo86) [reply website/layouts/partials/scripts.html:21]: Just this nit. Otherwise, all fine here.

File: website/layouts/partials/head.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+{{ if eq (getenv "HUGO_ENV") "production" }}
+<META NAME="ROBOTS" CONTENT="INDEX, FOLLOW">
+{{ else }}
+<META NAME="ROBOTS" CONTENT="NOINDEX, NOFOLLOW">
+{{ end }}
+{{ range .AlternativeOutputFormats -}}
+<link rel="{{ .Rel }}" type="{{ .MediaType.Type }}" href="{{ .Permalink | safeURL }}">
+{{ end -}}
+{{ partialCached "favicons.html" . }}
+<title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
```
Comment (tengqm) [inline website/layouts/partials/head.html:14]: Just out of curiosity, why additional "with" stanza here?

```suggestion
<title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ .Title }} | {{ .Site.Title }}{{ end }}</title>
```

Comment (matthewcarleton) [reply website/layouts/partials/head.html:14]: If I remember correctly this would be how Hugo ships it. I just reused their example.

**Comment by camilamacedo86** [comment]
Hi @matthewcarleton, 

What is missing here for we get this one merged?

**Comment by matthewcarleton** [comment]
> Hi @matthewcarleton,
> 
> What is missing here for we get this one merged?

I'm not sure :) If @tengqm is ok with it I'd say merge it

**Comment by camilamacedo86** [comment]
Hi @matthewcarleton, 

Shows that @tengqm did not answer for a while. If you got more one approval, I think it is ok to move forward since it is a very small change and we can do a follow up if required. @jmrodri @asmacdo wdyt? Are you ok with this change as well?

**Comment by tengqm** [comment]
LGTM

**Comment by camilamacedo86** [comment]
Hi @matthewcarleton,  

It has 3 lgtm so I am going forward and getting it merged.

**Pull request merged**